app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -976,26 +985,22 @@ public void onResponse(Call<SusiResponse> call,
                                 if (response != null && response.isSuccessful() && response.body() != null) {
                                     final SusiResponse susiResponse = response.body();
 
-                                    parseSusiResponse(susiResponse);
+                                    int actionSize = response.body().getAnswers().get(0).getActions().size();
 
-                                    realm.executeTransactionAsync(new Realm.Transaction() {
```

Comment:
No need of adding these changes here. Same changes has already been done in PR #681 . So, Its better to remove them now rather than solving merge conflicts later. Just keep the changes in parseSusiResponses() method and remove the rest of changes in MainActivity.